### PR TITLE
Milestone 1: The frontend page of Newsweek site is ready

### PR DESCRIPTION
**Pull request by RaminMammadzada**
I finished the following parts:
- Designed banner part.
- Designed header navbar.
- Designed the main part of the page.
- Designed the footer part of the page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,21 +5,2684 @@
...
+        <div class="row align-items-baseline">
+          <!-- picked 1 -->
+          <div class="my-1 col-lg-3 col-md-3">
+            <div class="card my-1">
+              <a href="#">
+                <img
+                  class="card-img-top position-relative img-fluid"
+                  src="https://d.newsweek.com/en/full/1604708/down-jackson.webp?w=480&h=320&l=63&t=51&q=90&f=6aa994b38cbdb4a0635bc48065702d5a"
+                  alt="picture"
+                />
```
Comment (elmejdki) [inline index.html]: on tablet view, the editor's pick should ave two articles in a row, not four:

| Source Page	| Clone Page |
|:---:|:---:|
| image | image |

**Review by elmejdki — CHANGES_REQUESTED** [review]
# Status: required changes :recycle: 
## Great job so far @RaminMammadzada you really impressed me with your clone :100: :+1: :clap:  , but in order to make your clone approved you need to do some changes :hammer: :wrench: :toolbox: 

- [x] on the top story section on the mobile view you have the first article goes out from its container, please fix it, also the navbar should go to the extreme on mobile and tablet views, remove the margins please:

| Source Page	| Clone Page |
|:---:|:---:|
| image | image |

- [x] Please fix the subscribe section on mobile view:

| Source Page	| Clone Page |
|:---:|:---:|
| image | image |

- [x] also the newsletter section:

| Source Page	| Clone Page |
|:---:|:---:|
| image | image |

### Please make all the required changes before you submit another code review.
## Happy Coding :tophat:

File: index.html (hunk 1 above)
Comment (RaminMammadzada) [reply index.html]: Thank you. It is done now.

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Changes required 🛑

## Hi @RaminMammadzada 👋 . You are almost there. 

### There are some crucial changes you need to apply.

#### Bootstrap and font-awesome files are an integral part of this project. This is how your page looks right now:

image

Remove these from the `.gitignore` and make a commit. These files are required not only in development but also in production too.

image


Make this change and request another review.

Happy coding 👨‍💻

File: index.html [hunk 2]
```diff
@@ -13,13 +13,2689 @@
...
     <link rel="stylesheet" href="assets/css/custom.css" />
+
     <title>Newsweek</title>
   </head>
 
-  <body>
-    <header></header>
-    <main></main>
-    <footer></footer>
+  <body class="container-fluid px-0">
```
Comment (tirthajyoti-ghosh) [inline index.html]: Good job making the clone full width in smaller screens but you still need to put spacing on either side of your clone in larger screens(desktop).

| Source Page	| Clone Page |
|:---:|:---:|
|image|image|

File: index.html [hunk 3]
```diff
@@ -13,13 +13,2689 @@
...
+                </h5>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <!-- subscribe and newsletter -->
+      <div class="container-fluid row table">
+        <div class="col-lg-8 d-flex flex-column">
```
Comment (tirthajyoti-ghosh) [inline index.html:2458]: The magazine image is leaking out of its container.

| Source Page	| Clone Page |
|:---:|:---:|
|image|Annotation 2020-07-11 155039|

File: index.html [hunk 4]
```diff
@@ -13,13 +13,2689 @@
...
+
     <title>Newsweek</title>
   </head>
 
-  <body>
-    <header></header>
-    <main></main>
-    <footer></footer>
+  <body class="container-fluid px-0">
+    <header class="container-fluid max-auto mx-0 px-0">
```
Comment (tirthajyoti-ghosh) [inline index.html]: Please make the `header` fixed in tablet and mobile screens.

| Source Page	| Clone Page |
|:---:|:---:|
|Annotation 2020-07-11 160158|Annotation 2020-07-11 160222|

File: index.html [hunk 5]
```diff
@@ -13,13 +13,2689 @@
...
+          <!-- /magazine 1-->
+          <!-- magazine 2-->
+          <div class="my-1 col-lg-3 col-md-3">
+            <div class="card my-1">
+              <a href="#">
+                <img
+                  class="card-img-top position-relative img-fluid"
+                  src="https://d.newsweek.com/en/full/1604216/good-list-banner.webp?w=466&h=311&q=90&f=2062901c3efd89779019feae0efa97da"
+                  alt="picture"
+                />
```
Comment (tirthajyoti-ghosh) [inline index.html:1397]: #### [OPTIONAL]

Consider increasing the height of the images in `in the magazine` section.

| Source Page	| Clone Page |
|:---:|:---:|
|Annotation 2020-07-11 160039|Annotation 2020-07-11 155926|

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Changes required 🛑

## It's me again 😄. Great job fixing the previous issue. You are almost there.

### Please check the comments and make the necessary changes.

Happy coding 👨‍💻

File: index.html (hunk 2 above)
Comment (RaminMammadzada) [reply index.html]: Done.

File: index.html (hunk 3 above)
Comment (RaminMammadzada) [reply index.html:2458]: Done.

**Comment by bertil291utn** [comment]
**Status:** Changes requested 🔴  :shipit: ? Not yet

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

Your work is amazing you master it GitHub Flow you have a professional README file. 

**Small screen**

- [x] After **OPINION** section you are missing to add **MORE STORIES, MY TURN and CULTURAL & TRAVEL**  ones

|                     **ACTUAL WEB PAGE**                      |                            YOURS                             |
| :----------------------------------------------------------: | :----------------------------------------------------------: |
| image | image |

**Large screen**

- [x] Remove extra padding 

image

- [x] Weather link and search input should be with the main nav from 1024px

|                     **ACTUAL WEB PAGE**                      |                            YOURS                             |
| :----------------------------------------------------------: | :----------------------------------------------------------: |
| image | image |



<hr>

Guys work in the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

**Comment by RaminMammadzada** [comment]
> **Status:** Changes requested 🔴 :shipit: ? Not yet
> 
> # Hey guys, you've been doing a great job, and also I hope you enjoyed coding.
> Your work is amazing you master it GitHub Flow you have a professional README file.
> 
> **Small screen**
> 
> * [ ]  After **OPINION** section you are missing to add **MORE STORIES, MY TURN and CULTURAL & TRAVEL**  ones
> 
> **ACTUAL WEB PAGE**	YOURS
> image	image
> **Large screen**
> 
> * [ ]  Remove extra padding
> 
> image
> 
> * [ ]  Weather link and search input should be with the main nav from 1024px
> 
> **ACTUAL WEB PAGE**	YOURS
> image	image
> Guys work in the requested changes above and request another review.
> Good luck.
> 
> > “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”


In the large screen, there should be padding in the sides. Please check the review are made previously by other people before reviewing the project. In this point, your review doesn't correlate with tirthajyoti-ghosh's review.

**Review by expjazz — APPROVED** [review]
## Status: Project Approved :green_circle: 

#### Congratulations on this project :clap: 

#### You implemented all the requested changes, so you can merge now. I will suggest just one small change, but at this point, it is optional (only if you want to make your project looks more like the original than already is). 
- Use the original color for the background (yours is white, original is a light tone of blue).

Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Pull request merged**